docs/ingestion/native-batch-input-source.md

```diff
@@ -29,7 +29,7 @@ For general information on native batch indexing and parallel task indexing, see
 
 ## S3 input source
 
-> You need to include the [`druid-s3-extensions`](../development/extensions-core/s3.md) as an extension to use the S3 input source.
+> You need to include the [`druid-s3-extensions`](../development/extensions-core/s3.md) as an extension to use the S3 input source. 
```

Comment:
This change is a typo right? There is no need for the trailing space.

Reply:
Yeah, the trailing space was removed in the original PR. Reverting that commit would have brought back this change.